src/main/scala/io/citrine/lolo/transformers/Standardizer.scala

```diff
@@ -184,6 +184,8 @@ class StandardizerPrediction[T](baseResult: PredictionResult[T], trans: Seq[Opti
       case None => None
       case Some(x) =>
         Some(x.map(g => g.zip(trans.tail).map {
+          // Return 0.0 if trying to undo inverse of 0.0
+          case (0.0, Some((_, Double.PositiveInfinity))) => 0
```

Comment:
Having an affine transformation with infinite stretch is probably wrong to begin with.  Could we try to catch it earlier (e.g in the calculation of `trans`)?